plugins/processors/template/README.md

```diff
@@ -59,4 +59,32 @@ Add the year as a tag, similar to the date processor:
   template = '{{.Time.UTC.Year}}'
 ```
 
+Add all fields as a tag:
+
+```toml
+[[processors.template]]
+  tag = "message"
+  template = 'Fields: {{ .FieldList }}'
+  # Advanced example:
+  # template 'Fields: {{ range $name, $value := .FieldList }}{{$name}}:{{$value}} {{end}}'
+```
+
+```diff
+- cpu,hostname=localhost time_idle=42
++ cpu,hostname=localhost,message=Fields:\ map[time_idle:42] time_idle=42
```

Comment:
Is the message value here interpreted as a string by line protocol?

Reply:
Tags are always only strings. I'm just pasting what the line protocol serialiser is outputting

Reply:
Is there a different example that we can use then? This particular example has me concerned for two reasons:

1. Having a string with what is Golang map's referenced in them seems not useful at all. Is there a use-case I'm missing?
2. If someone doesn't post-process this it is going to blow up their cardinality

Reply:
1. All output plugins that expect a `message` tag or field (syslog, groundwork, ..) where you want to list all metric fields for example. Another example could also be by doing something like `{{ range $field, $value := .FieldList }}`, but I thought that would be too advanced as example?
2. Sure, but InfluxDB is not the only type of output available..

Reply:
> 1. All output plugins that expect a `message` tag or field (syslog, groundwork, ..) where you want to list all metric fields for example. See also my commented out example, which doesn't include 'map'.
> 

I still do not understand what the use case is for that.

>     2. Sure, but InfluxDB is not the only type of output available.. (While thinking of it, wouldn't it be nice to also have option to directly put result to a field? That would make more sense in the examples)

I understand that we have more than InfluxDB, however, we know users rely on these examples and you cannot be certain a user of InfluxDB starts using your example and then complains when their cardinality blows up.